src/components/Checkbox.vue

```diff
@@ -54,6 +54,8 @@ export default {
     clicked(event) {
       if (this.disabled) {
         event.preventDefault();
+      } else {
+        this.$emit('input', !this.value);
```

Comment:
Ah, `this.value` here is the (old) value, _not_ the state from the HTML element, right?  Coding it like this seems likely to end up with state drift, but I guess that's how it works… And the code in dashboard does the same thing (just more complicated)…

Reply:
Yes. I don't know why they don't use `event.target.value` -- would be clearer.

And the code in dashboard is more complicated because it uses jquery. Once PR #77 is merged in I'll see if I can pull in the original `Checkbox.vue` from dashboard. I'd like to add a tooltip when the checkbox is disabled, but we'll need to implement L18n with the `t` function to make that happen.